WordPress/src/main/java/org/wordpress/android/ui/reader/views/ReaderPostDetailHeaderView.kt

```diff
@@ -153,45 +138,8 @@ class ReaderPostDetailHeaderView @JvmOverloads constructor(
 
         fun updateInteractionSection(state: InteractionSectionUiState, readingPreferences: ReaderReadingPreferences?)
 
-        class ImprovementsDisabled(
-            private val binding: ReaderPostDetailHeaderViewBinding,
-            private val uiHelpers: UiHelpers,
-        ) : ReaderPostDetailHeaderBinding {
-            override val titleText: MaterialTextView
-                get() = binding.textTitle
-            override val blogNameText: MaterialTextView
-                get() = binding.layoutBlogSection.textAuthorAndBlogName
-            override val blogUrlText: MaterialTextView
-                get() = binding.layoutBlogSection.textBlogUrl
-            override val expandableTagsView: ReaderExpandableTagsView
-                get() = binding.expandableTagsView
-            override val blogAvatarImage: ImageView
-                get() = binding.layoutBlogSection.imageAvatarOrBlavatar
-            override val authorAvatarImage: ImageView
-                get() = binding.layoutBlogSection.authorsAvatar
-            override val followButton: ReaderFollowButton
-                get() = binding.headerFollowButton
-            override val blogSectionRoot: ViewGroup
-                get() = binding.layoutBlogSection.root
-
-            override fun setAuthorAndDate(authorName: String?, dateLine: String) = with(binding) {
-                uiHelpers.setTextOrHide(textAuthor, authorName)
-                uiHelpers.setTextOrHide(postDetailTextDateline, dateLine)
-
-                textBy.setVisible(authorName != null)
-                postDetailDotSeparator.setVisible(authorName != null)
-            }
-
-            override fun updateInteractionSection(
-                state: InteractionSectionUiState,
-                readingPreferences: ReaderReadingPreferences?
-            ) {
-                // do nothing
-            }
-        }
-
         class ImprovementsEnabled(
```

Comment:
Now that we have just one implementation, what do you think about making `ReaderPostDetailHeaderBinding` a concrete class and moving the implementation from this subclass to there?